tests/crashes/auxiliary/aux132985.rs

```diff
@@ -0,0 +1,6 @@
+#![feature(adt_const_params)]
```

Comment:
apparently `crashes` tests dont support aux build, can either wait for #133013 or just remove this test and land without it since we're tracking it in an issue anyway :woman_shrugging: